src/v2/view-builder/internals/BaseForm.js

```diff
@@ -45,8 +45,8 @@ export default Form.extend({
   },
 
   handleClearFormError() {
-    const formErrorContainer = this.$('.o-form-error-container');
-    formErrorContainer.empty();
+    const formErrorContainer = this.$('.o-form-error-container .infobox-error');
+    formErrorContainer.remove();
```

Comment:
Is it possible now for duplicate warning messages to build up if the form fails to save multiple times?

Reply:
Good point. For resend warning messages they won't pile up, because we hide/show them.. But messages other than errors (i.e INFO) might.

Exploring less disruptive fix.